apps/portals/adknowledgeportal/src/config/synapseConfigs/computational_tools.ts

```diff
@@ -1,17 +1,17 @@
 import type {
   CardConfiguration,
-  GenericCardSchema,
   QueryWrapperPlotNavProps,
 } from 'synapse-react-client'
 import { SynapseConstants } from 'synapse-react-client'
+import { TableToGenericCardMapping } from 'synapse-react-client/components/GenericCard/TableRowGenericCard'
 import { computationalSql } from '../resources'
 import { PROGRAM_TABLE_COLUMN_NAMES } from './programs'
 
 export const COMPUTATIONAL_TOOLS_COLUMN_NAMES = {
   GRANT: 'grant',
 }
 
-const computationalSchema: GenericCardSchema = {
+const computationalSchema: TableToGenericCardMapping = {
```

Comment:
Renamed `GenericCardSchema` to `TableToGenericCardMapping` since this type defines the mapping between table data and the GenericCard UI